src/ia-dropdown.ts

```diff
@@ -0,0 +1,286 @@
+    const { label, url = undefined, id } = availableOption;
+    let component;
+    const selected = this.selectedOption === id ? 'selected' : '';
+
+    if (url) {
+      component = html`<a
+        href=${url}
+        @click=${() => this.optionClicked(availableOption)}
+        >${label}</a
+      >`;
+    } else {
+      component = html`<button
+        @click=${() => this.optionClicked(availableOption)}
+      >
+        ${label}
+      </button>`;
+    }
+
+    return html`<li class=${selected}>${component}</li>`;
+  }
+
+  optionClicked(option: optionInterface): void {
+    this.selectedOption = option.id;
+
+    this.dispatchEvent(
+      new CustomEvent('optionSelected', {
+        detail: { option },
+      })
+    );
+
+    if (option.selectedHandler) {
+      option?.selectedHandler(option);
+    }
+  }
+
+  toggleOptions() {
+    if (!this.open) {
+      this.open = true;
+      return;
+    }
+
+    this.open = false;
+  }
+
+  get caret(): SVGTemplateResult {
+    if (!this.open) {
+      return this.caretDown;
+    }
+
+    return this.caretUp;
+  }
+
+  get dropdownState(): string {
+    if (this.open) {
+      return 'open';
+    }
+
+    return 'closed';
+  }
+
+  get caretUp(): SVGTemplateResult {
+    return svg`<svg class="caret-up-svg" height="4" viewBox="0 0 8 4" width="8" xmlns="http://www.w3.org/2000/svg">
+    <path d="m6.7226499 3.51689722c.22976435.15317623.54019902.0910893.69337525-.13867505.13615665-.20423497.10222882-.47220946-.06836249-.63681849l-.07031256-.05655675-3.2773501-2.18490007-3.2773501 2.18490007c-.22976434.15317623-.29185128.4636109-.13867505.69337524.13615665.20423498.39656688.27598409.61412572.18182636l.07924953-.04315131 2.7226499-1.81402514z"
+      fill=""></path>
+  </svg>`;
+  }
+
+  get caretDown(): SVGTemplateResult {
+    return svg`<svg class="caret-down-svg" height="4" viewBox="0 0 8 4" width="8" xmlns="http://www.w3.org/2000/svg">
+    <path d="m6.7226499.58397485c.22976435-.15317623.54019902-.09108929.69337525.13867505.13615665.20423498.10222882.47220947-.06836249.63681849l-.07031256.05655676-3.2773501 2.18490006-3.2773501-2.18490006c-.22976434-.15317623-.29185128-.4636109-.13867505-.69337525.13615665-.20423497.39656688-.27598409.61412572-.18182636l.07924953.04315131 2.7226499 1.81402515z"
+    fill=""></path>
+  </svg>`;
+  }
+
+  render() {
+    return html`
+      <div class="ia-dropdown-group">
+        <button @click=${this.toggleOptions} class="click-main">
+          <span class="cta sr-only">Toggle ${this.optionGroup}</span>
+          <slot name="dropdown-label"></slot>
+          ${this.displayCaret
+            ? html`<span class="caret">${this.caret}</span>`
+            : nothing}
+        </button>
+
+        <ul class="dropdown-main ${this.dropdownState}">
+          ${this.options.map(o => this.renderOption(o))}
+        </ul>
+      </div>
+    `;
+  }
+
+  static styles = css`
+    :host {
+      display: inline;
+      font: sanserif;
```

Comment:
Is this font familty match with our parent font? why we need to define this font separately  ?

Reply:
thank you for catching! removed